### ISSUE TITLE
V8 CI not working

### ISSUE THREAD

**Issue by MylesBorins** [nodejs/build#1940]
ppcle-ubuntu1404  and centos7-ppcle  have both failed the last three jobs with the error

```
10:31:23 + ln -s /usr/bin/ccache ppc64le-redhat-linux-g++ /home/iojs/build-tools/g++
10:31:23 ln: target ‘/home/iojs/build-tools/g++’ is not a directory
```

**Comment by sam-github** [comment]
@MylesBorins a specific CI job name would help me fix this.

This must have been triggered by https://github.com/nodejs/build/pull/1927

Some CI job is doing `ln -s $CXX /some/path`, without considering that CXX might be `CXX="ccache g++"`

**Comment by MylesBorins** [comment]
seeing it in the following jobs

https://ci.nodejs.org/job/node-test-commit-v8-linux/2528/
https://ci.nodejs.org/job/node-test-commit-v8-linux/2529/
https://ci.nodejs.org/job/node-test-commit-v8-linux/2530/
https://ci.nodejs.org/job/node-test-commit-v8-linux/2531/

**Comment by sam-github** [comment]
Ok, found the broken code, its in https://github.com/nodejs/node/blob/3473e58fee153a7fee143a33c512ee9cda08d20a/tools/make-v8.sh#L15-L23

I am looking at fixing it.

**Comment by MylesBorins** [comment]
@sam-github odd that this was not broken the last time we ran this CI... any idea why this commit from July is just starting to break now?

**Comment by sam-github** [comment]
yes, because https://github.com/nodejs/build/issues/1940#issuecomment-537102717

**Comment by MylesBorins** [comment]
@sam-github OHHHHHH I missed that... thanks for digging in!

**Comment by sam-github** [comment]
This is going slowly, I can't do v8 builds, even in the jenkins workspace. Apparently, too much of the necessary setup is known only to the the jekins jobs.
```
+ tools/node/fetch_deps.py .
Using depot tools in /home/iojs/build/workspace/node-test-commit-v8-linux/nodes/centos7-ppcle/v8test/v8test/deps/v8/_depot_tools                                             
Initializing temporary git repository in v8.
Fetching dependencies.
Syncing projects: 100% (29/29), done.

________ running 'vpython v8/third_party/depot_tools/update_depot_tools_toggle.py --disable' in '.'                                                                          
[E2019-10-01T20:16:22.715674Z 23649 0 annotate.go:241] goroutine 1:
[E2019-10-01T20:16:22.715734Z 23649 0 annotate.go:241] #0 vpython/pep425.go:258 - main.getPEP425CIPDTemplateForTag()                                                         
[E2019-10-01T20:16:22.715747Z 23649 0 annotate.go:241]   reason: failed to infer CIPD platform for tag [python:"cp27" abi:"cp27mu" platform:"linux_ppc64le" ]                
[E2019-10-01T20:16:22.715760Z 23649 0 annotate.go:241]
[E2019-10-01T20:16:22.715771Z 23649 0 annotate.go:241] #1 vpython/main.go:50 - main.glob..func1()                                                                            
[E2019-10-01T20:16:22.715784Z 23649 0 annotate.go:241] #2 go.chromium.org/luci/vpython/cipd/cipd.go:278 - cipd.(*PackageLoader).expanderForTags()                            
[E2019-10-01T20:16:22.715797Z 23649 0 annotate.go:241]   reason: failed to get CIPD template arguments                                                                       
[E2019-10-01T20:16:22.715808Z 23649 0 annotate.go:241]
[E2019-10-01T20:16:22.715819Z 23649 0 annotate.go:241] #3 go.chromium.org/luci/vpython/cipd/cipd.go:68 - cipd.(*PackageLoader).Resolve()                                     
[E2019-10-01T20:16:22.715831Z 23649 0 annotate.go:241] #4 go.chromium.org/luci/vpython/venv/config.go:182 - venv.(*Config).makeEnv()                                         
[E2019-10-01T20:16:22.715844Z 23649 0 annotate.go:241]   reason: failed to resolve packages                                                                                  
[E2019-10-01T20:16:22.715855Z 23649 0 annotate.go:241]
[E2019-10-01T20:16:22.715866Z 23649 0 annotate.go:241] #5 go.chromium.org/luci/vpython/venv/venv.go:163 - venv.With()                                                        
[E2019-10-01T20:16:22.715878Z 23649 0 annotate.go:241] #6 go.chromium.org/luci/vpython/run.go:62 - vpython.Run()                                                             
[E2019-10-01T20:16:22.715889Z 23649 0 annotate.go:241] #7 go.chromium.org/luci/vpython/application/application.go:312 - application.(*application).mainImpl()                
[E2019-10-01T20:16:22.715901Z 23649 0 annotate.go:241] #8 go.chromium.org/luci/vpython/application/application.go:397 - application.(*Config).Main.func1()                   
[E2019-10-01T20:16:22.715912Z 23649 0 annotate.go:241] #9 go.chromium.org/luci/vpython/application/support.go:46 - application.run()                                         
[E2019-10-01T20:16:22.715924Z 23649 0 annotate.go:241] #10 go.chromium.org/luci/vpython/application/application.go:396 - application.(*Config).Main()                        
[E2019-10-01T20:16:22.715935Z 23649 0 annotate.go:241] #11 vpython/main.go:105 - main.mainImpl()                                                                             
[E2019-10-01T20:16:22.715947Z 23649 0 annotate.go:241] #12 vpython/main.go:111 - main.main()                                                                                 
[E2019-10-01T20:16:22.715958Z 23649 0 annotate.go:241] #13 runtime/proc.go:200 - runtime.main()                                                                              
[E2019-10-01T20:16:22.715970Z 23649 0 annotate.go:241] #14 runtime/asm_ppc64x.s:857 - runtime.goexit()                                                                       
Error: Command 'vpython v8/third_party/depot_tools/update_depot_tools_toggle.py --disable' returned non-zero exit status 1 in .                                              
Uninitializing temporary git repository
>> Cleaning up /home/iojs/build/workspace/node-test-commit-v8-linux/nodes/centos7-ppcle/v8test/v8test/deps/v8/.git                                                           
Traceback (most recent call last):
  File "tools/node/fetch_deps.py", line 101, in <module>
    FetchDeps(sys.argv[1])
  File "tools/node/fetch_deps.py", line 85, in FetchDeps
    env=env)
  File "/usr/lib64/python2.7/subprocess.py", line 542, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/bin/python', '/home/iojs/build/workspace/node-test-commit-v8-linux/nodes/centos7-ppcle/v8test/v8test/deps/v8/_depot_tools/gclient.py', 'sync', '--spec', "solutions = [{'url': 'https://chromium.googlesource.com/v8/v8.git', 'managed': False, 'name': 'v8', 'deps_file': 'DEPS', 'custom_deps': {'v8/third_party/qemu-linux-x64': None, 'v8/third_party/catapult': None, 'v8/test/benchmarks/data': None, 'v8/test/mozilla/data': None, 'v8/third_party/colorama/src': None, 'v8/testing/gmock': None, 'v8/test/test262/data': None, 'v8/third_party/android_sdk': None, 'v8/test/test262/harness': None, 'v8/tools/swarming_client': None, 'v8/third_party/fuchsia-sdk': None, 'v8/third_party/googletest/src': None, 'v8/base/trace_event/common': None, 'v8/tools/luci-go': None, 'v8/third_party/instrumented_libraries': None, 'v8/third_party/android_ndk': None}}]"]' returned non-zero exit status 2
Makefile:292: recipe for target 'v8' failed
make: *** [v8] Error 1
```

I'll revert the change that triggered this for now.

**Comment by sam-github** [comment]
Fixed by f54e6ceea58131a2e9b2892ed8f6391f4563f213 and 7b9805aaa19235514e2ec1e70bcca7d882798175

ci: https://ci.nodejs.org/job/node-test-commit-v8-linux/2533/

**Comment by sam-github** [comment]
green: https://ci.nodejs.org/job/node-test-commit-v8-linux/2533/
